.github/workflows/ci.yaml

```diff
@@ -175,3 +174,26 @@ jobs:
         run: |
           unity -quit -batchmode -nographics -logFile - -returnlicense | Out-Default
         if: ${{ always() }}
+
+  pack:
+    needs: build
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+        with:
+          submodules: recursive
+
+      - name: Prepare Sentry package for release
+        run: |
+          New-Item "package-release" -ItemType Directory
+          Copy-Item "package-dev/*" -Destination "package-release/" -Exclude "Tests", "Tests.meta", "package.json" -Recurse
```

Comment:
Is this grabbing the files that were written here in the workflow above? Doesn't seem like these are related builds

Reply:
yeah i didn't consider that it puts something there 😅